packages/e2e-tests/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "test:e2e": "run-s test:setup test:jest",
+    "test": "run-s test:setup test:jest",
```

Comment:
If we have the `test` command here, won't running `yarn test` in the workspace root try to run the E2E tests? I am worried that this is gonna be annoying if you don't have an auth token configured.

Reply:
Good point, thanks. My main motivation for renaming this to `test` was to still make `yarn test` execute all tests in the repo. But I agree that this is probably annoying for people to set up. I'll revert this change and add a top-level `test:all` command that includes E2E tests.

Reply:
Update: I opted for another solution because what I described above required a few not so nice hacks and I feel like `test` should always be the main test execution command. Here's what I'm gonna do:
* `yarn test`
  * top level: executes all tests **except** for E2E tests
  * package level: executes the tests of the package (also in E2E tests package)
* `yarn test:all`
  * top level: executes all tests, **including** E2E tests
  * package level: N/A